src/Processors/Formats/Impl/CHColumnToArrowColumn.cpp

```diff
@@ -697,6 +576,158 @@ namespace DB
         checkStatus(status, write_column->getName(), format_name);
     }
 
+    template <typename ColumnType>
+    static void fillArrowArrayWithBigIntegerColumnData(
+        ColumnPtr write_column,
+        const PaddedPODArray<UInt8> * null_bytemap,
+        const String & format_name,
+        arrow::ArrayBuilder* array_builder,
+        size_t start,
+        size_t end)
+    {
+        const auto & internal_column = assert_cast<const ColumnType &>(*write_column);
+        const auto & internal_data = internal_column.getData();
+        size_t fixed_length = sizeof(typename ColumnType::ValueType);
+        arrow::FixedSizeBinaryBuilder & builder = assert_cast<arrow::FixedSizeBinaryBuilder &>(*array_builder);
+        arrow::Status status;
```

Comment:
why not just remove declaration and created where it is assigned? I do not see it being used before being assigned in line 598